docs/source/models/gbd2017/causes/neural_tube_defects/index.rst

```diff
@@ -97,29 +104,143 @@ The Neural Tube Defects cause has 85 sequelae, which can be summarized as follow
+the other and permanently stays in that state. **Thus, incidence and remission
+rates are zero.**
 
-People born into the simulation should enter the "with-condition" state
-according to the birth prevalence of neural tube defects; if this data is
-unavailable, we can estimate the birth prevalence using the prevalence and
-mortality rates in the early neonatal age group.
+Data Description
+----------------
 
 .. todo::
 
     Look into data sources. In particular, is birth prevalence data available,
-    or do we need to estimate it using prevalence among the early neonatal age
-    group?
+    or do we need to estimate it using prevalence and mortality rates among the
+    early neonatal age group?
+
+
+State and Transition Data Tables
+++++++++++++++++++++++++++++++++
+
+.. list-table:: State Definitions
+   :widths: 1, 5, 10
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - C
+     - With **C**\ ondition
+     - Born with neural tube defects
+   * - F
+     - **F**\ ree of Condition
+     - Born without neural tube defects
+
+.. list-table:: State Data
+   :widths: 1, 5, 5, 10
+   :header-rows: 1
+
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - C
+     - prevalence
+     - prevalence_c642
+     -
+   * - C
+     - birth prevalence
+     - birth_prevalence_c642
+     -
+   * - C
+     - excess mortality rate
+     - :math:`\frac{\text{deaths_c642}}{\text{population} \,\times\, \text{prevalence_c642}}`
+     - = (cause-specific mortality rate) / prevalence
+   * - C
+     - disability weight
+     - :math:`\displaystyle{\sum_{s\in \text{sequelae_c642}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
+     - = average disability weight over all sequelae
+   * - F
+     - proportion of population
+     - 1 -- prevalence_c642
+     -
+   * - F
+     - proportion of births
+     - 1 -- birth_prevalence_c642
+     -
+   * - F
+     - excess mortality rate
+     - 0
+     -
+   * - F
+     - disability weight
+     - 0
+     -
+   * - All
+     - cause-specific mortality rate
+     - :math:`\frac{\text{deaths_c642}}{\text{population}}`
+     -
+
+.. list-table:: Transition Data
+   :widths: 1, 1, 1, 5, 10
+   :header-rows: 1
+
+   * - Transition
+     - Source State
+     - Sink State
+     - Value
+     - Notes
+   * - N/A
+     - N/A
+     - N/A
+     - N/A
+     - N/A
+
+.. list-table:: Data Sources and Definitions
+   :widths: 1, 3, 10
+   :header-rows: 1
+
+   * - Value
+     - Source
+     - Notes
+   * - prevalence_c642
+     - COMO
+     - Prevalence of neural tube defects
+   * - birth_prevalence_c642
+     - ???
+     - Birth prevalence of neural tube defects
+   * - deaths_c642
+     - CoDcorrect
+     - Deaths from neural tube defects
+   * - population
+     - Demography
+     - Mid-year population for given country
+   * - sequelae_c642
+     - gbd_mapping
+     - List of 85 sequelae for neural tube defects
+   * - prevalence_{`sid`}
```

Comment:
prevalence_s{id}?

Reply:
Hmm, yes, perhaps that makes more sense. Or maybe prevalence_s{sid} since 'id' could be ambiguous?